src/bindings/python/fluxacct/accounting/job_archive_interface.py

```diff
@@ -136,7 +136,7 @@ def add_job_records(rows):
     return job_records
 
 
-def view_job_records(conn, output_file, **kwargs):
+def view_job_records(conn, output_file, cuf=False, **kwargs):
```

Comment:
with the addition of this `cuf` parameter the functions logic gets a bit more confusing.  Perhaps it would it be worthwhile to split this function?  Perhaps something like `get_job_records()`and `output_job_records(job_records)`?  So there's no need for this extra parameter and the places that just need the job records returned can just call `get_job_records()`?

Reply:
Thanks for this suggestion. I agree, it would probably be best to split this up. I force-pushed a change to [a4b468a](https://github.com/flux-framework/flux-accounting/pull/211/commits/a4b468a5d82292874748430a29e2e7c619110ac9) that spits this function into two, `get_job_records()` and `output_job_records()`.

`get_job_records()` has pretty much all of the previous functionality of fetching job records from the job-archive DB, but does not print results or write them to an output file - it just returns a list of job records. This will be used by `calc_usage_factor()` when updating the job-usage values for every user/bank combo in the flux-accounting DB.

`output_job_records()` just calls `get_job_records()` and, based on the original argument `output_file`, decides whether to print the results to `stdout` or write them to an output file. Let me know what you think of this change.

Reply:
change LGTM